News/NewsPulling.py

```diff
@@ -11,14 +14,20 @@
 import sys
 
 class NewsPulling(object):
+
+
     """This class is used to pull news from the internet depending on the source specified """
     def __init__(self,newsSource):
         self.Source=newsSource
-        
-    def PullNews(self):
         Configuration = ConfigurationReader()
-        self.__APIKey=Configuration.GetAPIKEY()
-        self.__Limit=Configuration.GetLimit()
+        self.__APIKey = Configuration.GetAPIKEY()
+        self.__Limit = Configuration.GetLimit()
+
+
+    # Pulls news stories based on SOURCE -- self.Source will indicate the correctly formatted
+    # news source to be included in the URL
+    def PullNews(self):
+

```

Comment:
move the URL to the parameter of the class while stories or topic as input to the method based on which either it will fetch stories or topics. or if possible create an interface and then inherits different class both of which it will handle differently based on their use case.